tools/install.sh

```diff
@@ -192,18 +181,14 @@ if [ -d logs ]; then
 fi
 
 echo ""
-# Install external libraries using composer
-cd ..
-eval $composer_scr
 cd tools
 
-
 if [[ " ${debian[*]} " =~ " $os_distro " ]]; then
 echo "Ubuntu distribution detected."
     # for CentOS, the log directory is called httpd
     logdirectory=/var/log/apache2
     while true; do
-        read -p "Would you like to automatically create/install apache config files? (Works for Ubuntu 14.04 or later default Apache installations) [yn] " yn
+        read -p "Would you like to automatically set up your apache configuration files? [yn] " yn
```

Comment:
this is where the projectname should get asked, i think.   I can make an edit but it will be a big one. @kongtiaowang

Reply:
@christinerogers You are right.

Reply:
any updates on `$projectname` cleanup, @kongtiaowang ?